src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/ErrorConfiguration.kt

```diff
@@ -22,7 +22,11 @@ enum class Code {
   CONDITIONAL_FIELD_MUST_BE_SET,
   CANNOT_BE_EMPTY,
   CANNOT_BE_NEGATIVE_OR_ZERO,
-  INVALID_SERVICE_CATEGORY_FOR_CONTRACT
+  INVALID_SERVICE_CATEGORY_FOR_CONTRACT,
+  ADDRESS_MUST_BE_PROVIDED,
```

Comment:
feels like this is covered by `CANNOT_BE_EMPTY`

Reply:
I agree. I'll change to this.